pkgs/os-specific/linux/rtw88/default.nix

```diff
@@ -5,13 +5,13 @@ let
 in
 stdenv.mkDerivation {
   pname = "rtw88";
-  version = "unstable-2021-04-19";
+  version = "unstable-2022-05-08";
 
   src = fetchFromGitHub {
     owner = "lwfinger";
     repo = "rtw88";
-    rev = "0f3cc6a5973bc386d9cb542fc85a6ba027edff5d";
-    hash = "sha256-PRzWXC1lre8gt1GfVdnaG836f5YK57P9a8tG20yef0w=";
+    rev = "ce979583dff5fc2f6cbce354c3e2dceafee454ca";
```

Comment:
`dbcc43c31a4576f90e1e468d3a85c2dfdb25ec42` is the newest commit in rtw88 https://github.com/lwfinger/rtw88/commits/master

Reply:
Oh thanks for letting me know, and also for the review! Fixed.